nbclient/tests/test_execute.py

```diff
@@ -465,14 +465,14 @@ def process_message(self, msg, cell, cell_index):
         assert_notebooks_equal(original, executed)
 
     def test_execute_function(self):
-        # Test the executenb() convenience API
+        # Test the executb() convenience API
```

Comment:
`execute`

Reply:
Oooops. Fixed in ad4f1880a2d5c929b73f3cc294e252c734e22128.